extras/create_send_notifications.py

```diff
@@ -42,26 +42,49 @@ def create_output_directories():
         os.path.join(NOTIFICATIONS_BASE_DIR, NOTIFICATION_ARCHIVE_DIR)
     )
 
+def build_notifications_org_list(db):
+    """Return notifications organization list.
 
-def build_cyhy_org_list(db):
-    """Build list of CyHy organization IDs.
-
-    This is the list of CyHy organization IDs (and their descendants) that
-    receive CyHy reports.
+    This is the list of organization IDs that should
+    have a notification generated and sent.
+    """
+    notifications_to_generate = set()
+    cyhy_parent_ids = set()
+    ticket_owner_ids = db.NotificationDoc.collection.distinct("ticket_owner")
+    for request in db.RequestDoc.collection.find({"_id": {"$in": ticket_owner_ids}, "report_types": "CYHY"}, {"_id":1}):
+        # If the notification document's ticket owner has "CYHY" in their list of report_types,
+        # then a notification should be generated for that owner:
+        notifications_to_generate.add(request["_id"])
+        logging.debug("{} - Log request id".format(request["_id"]))
+        # Recursively check for any parents of the ticket owner that have "CYHY" in
```

Comment:
Since now we're checking for more than just parents:
```suggestion
        # Recursively check for any ancestors of the ticket owner that have "CYHY" in
```